src/main/resources/application.yaml

```diff
@@ -184,4 +184,6 @@ hmcts_service_id: BHA1
 launchdarkly:
   sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
   offline-mode: false
-  env: ${LAUNCH_DARKLY_ENV:default}
\ No newline at end of file
+  env: ${LAUNCH_DARKLY_ENV:default}
+
+bundlesClaimantSubmittedRespondentNotificationTemplateId: ${BUNDLES_CLAIMANT_SUBMITTED_RESPONDENT_NOTIFICATION_TEMPLATE_ID:5c829358-0b54-4335-9d60-1c8296c4a516}
```

Comment:
ON L89 the template variables begin. Please put this variable in that block.

Reply:
The variable is moved within the template block